sirepo/reply.py

```diff
@@ -396,6 +406,9 @@ def render_static_jinja(self, base, ext, j2_ctx, cache_ok=False):
             return r.headers_for_cache(path=p)
         return r.headers_for_no_cache()
 
+    def set_cookies_for_deletion(self, names_and_paths):
+        self._cookies_to_delete = names_and_paths
```

Comment:
It should assert that  self._cookies_to_delete is None. We might want multiple calls to it could potentially merge the two lists, but for now, I like the simpler is None check.

Reply:
https://github.com/radiasoft/sirepo/pull/5523/commits/711e56e4b6d7aae82470e59ce8f2c1e71199f412